services/bettersocial/fcmToken/sendChatNotification.js

```diff
@@ -15,7 +15,10 @@ const sendChatNotification = async (userTargetId, payload) => {
         };
         messaging()
           .send(newPayload)
-          .then(() => {});
+          .then(() => {})
+          .catch((e) => {
+            console.log('error send chat notification in promise', e.code);
+          });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While it's good that you're catching errors from the promise, using `console.log` for error handling is not a best practice. It would be better to use a proper logging library that can provide more context about the error and handle different log levels. Also, consider rethrowing the error after logging so it can be handled by an upper layer or at least fail loudly instead of failing silently.

```diff
-            console.log('error send chat notification in promise', e.code);
+            logger.error('error send chat notification in promise', { code: e.code });
+            throw e;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->